paddle/fluid/operators/CMakeLists.txt

```diff
@@ -6,6 +6,7 @@ unset(OP_LIBRARY CACHE)
 set(pybind_file ${PADDLE_BINARY_DIR}/paddle/fluid/pybind/pybind.h.tmp CACHE INTERNAL "pybind.h file")
 set(pybind_file_final ${PADDLE_BINARY_DIR}/paddle/fluid/pybind/pybind.h)
 file(WRITE ${pybind_file} "// Generated by the paddle/fluid/operators/CMakeLists.txt.  DO NOT EDIT!\n\n")
+file(WRITE ${pybind_file} "USE_OP_DEVICE_KERNEL(elementwise_add, NPU);\n")
```

Comment:
we should find a common solution for that.

Reply:
The common solution will add `grad op` incorrectly, which will cause compilation failure because `elementwise_add_grad_op  NPU` is not registered. Here is a temporary solution and some comments are added here.